spotify/Spotifier.py

```diff
@@ -0,0 +1,40 @@
+from syrics.api import Spotify
+from typing import List, Dict, Any
+
+class Spotifier():
+
+    # Initialize Syrics with Spotify's SP_DC Key
+
+    def __init__(self, SP_DC_KEY: str) -> None:
+        try:
+            self.__sp = Spotify(SP_DC_KEY)
+        except Exception as e:
+            print(f"Error Setting up Syrics's Spotify: {e}")
+            return 
+
+    # Functions
+
+    # Get and Set for SP_DC KEY
+
+    def set_sp_dc(self, SP_DC_KEY: str) -> None:
+        self.__sp.session.cookies.set('sp_dc', SP_DC_KEY)
+
+    def get_sp_dc(self) -> str:
+        return self.__sp.session.cookies.get('sp_dc')
+
+    # Function for getting Lyrics from Spotify Tracks 
+    # Input: Spotify Track Text
+    # Output: List of Dict
+    # Key: startTimeMs, words, syllables, and endTimeMs
+
+    def get_lyrics_str_from_spotify_track_id ( self, spotify_track_id: str ) -> List[Dict[str, Any]]:
+        try:
+            lyrics = self.__sp.get_lyrics(spotify_track_id)['lyrics']['lines']
+            return lyrics
+        except Exception as e:
+            print(f"Error fetching lyrics: {e}")
```

Comment:
```suggestion
            print(f"{e}")
```
I think editing error message may make it harder to search for solution for a specific error message we got